profilepage/profilepage.html

```diff
@@ -333,105 +159,54 @@
 				box-shadow: 1px 6px 15px -8px black;
 				">
 			</div>
-			<div class="followingCardsContainer">
-				<div class="followingCards">
-					<div>
-							<div class="followingCardsBanner"></div>
-								<div class="profilepic">
-									<img src="img/profile.png" height="90px">
-								<div class="cardMeta">
-									<div class="userMeta">
-										<span>Username</span>
-										<span>@handle</span>
-									</div>
-									<div><span class="biodata" style="font-size: 15px;">Lorem ipsum dolor sit amet, consectetur adipisicing elit</span></div>
-								</div>
-							</div>
-					</div>
-				</div>
+			
+			<div class="followingCardsContainer"></div>
 
-				<div class="followingCards">
-					<div>
-							<div class="followingCardsBanner"></div>
-								<div class="profilepic">
-									<img src="img/profile.png" height="90px">
-								<div class="cardMeta">
-									<div class="userMeta">
-										<span>Username</span>
-										<span>@handle</span>
-									</div>
-									<div><span class="biodata" style="font-size: 15px;">Lorem ipsum dolor sit amet, consectetur adipisicing elit</span></div>
-								</div>
-							</div>
-					</div>
-				</div>
+			</div>
+		</div>
 
-				<div class="followingCards">
-					<div>
-							<div class="followingCardsBanner"></div>
-								<div class="profilepic">
-									<img src="img/profile.png" height="90px">
-								<div class="cardMeta">
-									<div class="userMeta">
-										<span>Username</span>
-										<span>@handle</span>
-									</div>
-									<div><span class="biodata" style="font-size: 15px;">Lorem ipsum dolor sit amet, consectetur adipisicing elit</span></div>
-								</div>
-							</div>
-					</div>
+		<div class="modal">
+			<div class="modalContent">
+				<div>
+					<span>Edit your profile</span>
+				</div>
+				<form name="editProfileForm" id="editProfileForm">
+					<input type="name" name="fullname" class="editProfileInput" placeholder="desired name">
+					<input type="email" name="email" class="editProfileInput" placeholder="email address">
+					<input type="email" name="handle" class="editProfileInput" placeholder="desired handle">
+					<input type="email" name="location" class="editProfileInput" placeholder="location">
+					<input type="email" name="email" class="editProfileInput" placeholder="URL">
+					<div style="font-size: 15px;">Birthday: <input type="date" name="birthday" class="editProfileInput"></div>
+					<div style="font-size: 15px;">Cover image: <input type="file" name="birthday" class="editProfileInput"></div>
+					<div style="font-size: 15px;">Profile picture: <input type="file" name="birthday" class="editProfileInput"></div>
+				</form>
+				<div style="position: absolute; bottom: 10px; padding: 10px;">
+					<button onclick="closeEditProfile()">Cancel</button>
+					<button onclick="closeEditProfile()">Save</button>
```

Comment:
You have to perform the same validations and make an API call to save these updates.